pipeline/service/_topic_modeling.py

```diff
@@ -4,14 +4,13 @@
 import spacy
 from spacy.cli.download import download as spacy_download
 from gensim import corpora
-from gensim.models import LdaModel
+from gensim.models import LdaModel, LsiModel
```

Comment:
Why are we additionally using an LSI model?

is LSI not more for finding similarity within a coherent corpus? I mean we can use this to empirically verify the coherence of topics but this seems to be even further out of scope.